src/Microsoft.AspNet.Server.Kestrel/Networking/UvWriteReq.cs

```diff
@@ -81,6 +81,54 @@ public void Init(UvLoopHandle loop)
             }
         }
 
+        public unsafe void Write2(
```

Comment:
yeah, but didn't want to have one code path if it meant passing a null safehandle for the non-'2' case

plus this'll probably change a bit in the future as buffer stuff is profiled